modules/trusted-firmware-m/Kconfig

```diff
@@ -20,6 +20,17 @@ config TFM_BL2
 	bool
 	default n
 
+config TFM_SECURE_UART1
+	bool "Use UART1 for TF-M logging"
+	default y
+	depends on !UART_1_NRF_UARTE
+	depends on SOC_NRF5340_CPUAPP || SOC_NRF9160
+	help
+	  Reserve the UART1 port for TF-M logging. This makes the UART1
+	  peripheral unavailable to the non-secure application. When this
+	  option is selected the device tree node for UART1 needs to be
+	  disabled for the non-secure application.
```

Comment:
> When this option is selected the device tree node for UART1 needs to be
> disabled for the non-secure application.

if this has a dependency to devicetree setting, then I think we should have a dependency in Kconfig.
Perhaps something like:
```
config TFM_SECURE_UART1
	bool "Use UART1 for TF-M logging"
	...
	depends on !"$(dt_nodelabel_enabled,uart1)"
```

Reply:
Added that one as well